.travis.yml

```diff
@@ -1,4 +1,35 @@
-language: objective-c
-script: 'curl -s https://raw.githubusercontent.com/atom/ci/master/build-package.sh | sh'
 notifications:
-  email: false
+  email:
+    on_success: never
+    on_failure: change
+
+script: 'curl -s https://raw.githubusercontent.com/atom/ci/master/build-package.sh | sh'
+
+git:
+  depth: 10
+
+sudo: false
+
+os:
+  - linux
+  - osx
+
+env:
+  global:
```

Comment:
This section isn't actually necessary unless you are specifying test packages (which `atom-package-deps` should be handling for you :wink:).